etc/profile-a-l/joplin.profile

```diff
@@ -0,0 +1,82 @@
+# Firejail profile for joplin
+# Description: A note taking and to-do application
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include joplin.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.config/joplin
+noblacklist ${HOME}/.config/nano
+noblacklist ${HOME}/.emacs
+noblacklist ${HOME}/.emacs.d
+noblacklist ${HOME}/.nanorc
+noblacklist ${HOME}/.vim
+noblacklist ${HOME}/.vimrc
+
+include allow-bin-sh.inc
+include allow-nodejs.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/joplin
+
+whitelist ${HOME}/.config/joplin
+whitelist ${HOME}/.config/nano
+whitelist ${HOME}/.emacs
+whitelist ${HOME}/.emacs.d
+whitelist ${HOME}/.nanorc
+whitelist ${HOME}/.vim
+whitelist ${HOME}/.vimrc
+
+include whitelist-common.inc
+include whitelist-runuser-common.inc
+include whitelist-var-common.inc
```

Comment:
whitelist-usr-share-common?

Reply:
that is because you need to whitelist /usr/share/joplin /usr/share/vim and so on

Reply:
yes, then whitelist them